### DISCUSSION TITLE
[Feature Request] Token Vault - Option to sort entries

### DISCUSSION THREAD

**Discussion by MorgothSauron** [standardnotes/forum#2544] [Suggestions] [answered]
It would be nice to have an option to sort the tokens by service name or account name. In the worst case I can still switch to the plain text editor, copy the content and sort it myself than paste back the new list. But it would be nice if the application could do it natively.

**Comment by atmoio** [comment]
Yeah, I'm sure we'll improve on the editor with features like this in the future.

**Comment by myreli** [comment, marked as answer]
Sorting by name is now available since https://github.com/standardnotes/token-vault/pull/51

<img width="792" alt="image" src="https://user-images.githubusercontent.com/17554234/218778831-9fb7618c-8355-46ad-97d8-8ed88184ca5b.png">
